src/Core/Content/Media/Message/UpdateThumbnailsMessage.php

```diff
@@ -2,20 +2,34 @@
 
 namespace Shopware\Core\Content\Media\Message;
 
+use Shopware\Core\Framework\Feature;
 use Shopware\Core\Framework\Log\Package;
 
 #[Package('discovery')]
```

Comment:
optional, to prevent the code coverage warnings. 

add a the `@codeCoverageIgnore` annotation to the class. as this is not really sensible to unit test this